std_msgs/Header header
string err_code
int32[4] data_good
float64[4] altitude_beam
float64[4] bottom_velocity_beam
float64[4] water_velocity_beam
float64[4] water_velocity_credit
float64[3] velocity_inst
int32 velocity_inst_flag
float64[3] velocity_earth
int32 velocity_earth_flag
float64[3] water_velocity_inst
int32 water_velocity_inst_flag
float64[3] water_velocity_earth
int32 water_velocity_earth_flag
float64 roll
float64 pitch
float64 heading
float64 altitude
float64 temperature
float64 pressure
float64 salinity
float64 sound_speed
string raw_data
